README.md

```diff
@@ -35,23 +35,41 @@ Extendable architecture
 - Inject your own implementations of key components (Json serializer, background task scheduler, http client factory)
 
 ## Getting started
-Install the latest version of `Unleash.Client` from [nuget.org](https://www.nuget.org/packages/Unleash.Client/).
+
+### Install the package
+
+Install the latest version of `Unleash.Client` from [nuget.org](https://www.nuget.org/packages/Unleash.Client/) or use the `dotnet` cli:
+
+``` bash
+dotnet add package unleash.client
+```
 
 ### Create a new a Unleash instance
 
-It is easy to get a new instance of Unleash. In your app you typically *just want one instance of Unleash*, and inject that where you need it. You will typically use a dependency injection frameworks to manage this. 
+---
+
+**⚠️ Important:** In almost every case, you only want a **single, shared instance of the Unleash  client (a *singleton*)** in your application . You would typically use a dependency injection framework to inject it where you need it. Having multiple instances of the client in your application could lead to inconsistencies and performance degradation.
+
+If you create more than 10 instances, Unleash will attempt to log warnings about your usage.
+
+---
+
+To create a new instance of Unleash you need to create and pass in an `UnleashSettings` object.
+
+When creating an instance of the Unleash client, you can choose to do it either **synchronously** or **asynchronously**.
+The SDK will synchronize with the Unleash API on initialization, so it can take a moment for the client to reach the correct state.
```

Comment:
These two lines aren't all that clear to me that they're talking about what happens when you're doing async init. Maybe something like: 

"When initialised asynchronously, the SDK will synchronize with the Unleash API on initialization, so it can take a moment for the client to reach the correct state.
This is usually not an issue and Unleash will do this in the background as soon as you initialize it."

Reply:
Hmm, it's not necessarily about async init, though. Just about what happens when you start it. Whether you wait for it or not is up to you 🤔  How about this:

```suggestion
The SDK will synchronize with the Unleash API on initialization, so it can take a moment for the it reach the correct state. With an asynchronous startup, this would happen in the background while the rest of your code keeps executing. In most cases, this isn't an issue. But if you want to **wait until the SDK is fully synchronized**, then you should use the configuration explained in the [synchronous startup](#synchronous-startup) section.
```

I'm a bit unsure about saying the "sdk will synchronize with the API", because I'm afraid it can be confusing when talking about synchronous and asynchronous startups (a lot of syncing here), so I'm open to rewording suggestions.

Reply:
Maybe I'm too close to this and I have too much context already but I think this is as clear as it gets. I think the rewording here is pretty solid so I'm happy to approve.

Small detail though: "so it can take a moment for the it reach the correct state" did you mean to remove the "the" here?

Reply:
D'ya mean the "to"? And no, I did not. added back in now 😅